src/stores/SchemaEditorStore.ts

```diff
@@ -11,7 +11,7 @@ export interface SchemaEditorState {
 }
 
 const useSchemaEditorStore = create<SchemaEditorState>(
```

Comment:
Should this file adopt the change in naming convention applied to the related component, _CatalogEditor.tsx_?